lite-node/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+//! LiteNode
+//!
+//! Application based on the [Abscissa] framework.
+//!
+//! [Abscissa]: https://github.com/iqlusioninc/abscissa
+
+// Tip: Deny warnings with `RUSTFLAGS="-D warnings"` environment variable in CI
+
+#![forbid(unsafe_code)]
+#![warn(
+    rust_2018_idioms,
+    trivial_casts,
+    unused_lifetimes,
+    unused_qualifications
+)]
+// TODO(ismail): add proper docs and remove this!
+#![allow(missing_docs)]
+
+pub mod application;
+pub mod commands;
+pub mod config;
+pub mod error;
+pub mod prelude;
```

Comment:
Everything below `prelude` should likely live somewhere else (in a different module)?